src/order/ideal.lean

```diff
@@ -93,6 +106,10 @@ instance : has_coe (ideal P) (set P) := ⟨carrier⟩
 /-- For the notation `x ∈ I`. -/
 instance : has_mem P (ideal P) := ⟨λ x I, x ∈ (I : set P)⟩
 
+lemma mem_def : x ∈ I ↔ x ∈ (I : set P) := iff_of_eq rfl
```

Comment:
If you flip the two sides of the `iff` and call this `mem_coe`, then I think it would be a good simp lemma.